yaraneba/spec/requests/task_request_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe 'Tasks', type: :request do
+  before do
+    @task = FactoryBot.create(:task)
+  end
```

Comment:
テストに依存しないので `let` の方がいいですね

```suggestion
  let(:task) { create(:task) }
```